ATM.py

```diff
@@ -30,31 +35,59 @@ def select(self):
                             'To withdraw money press 3\n'
                             'To change password press 4\n'
                             'for Exit press -1\n')
-        return self.choice
+        if self.choice == BALANCE_CHOICE:
+            balance = self.balance_dict[self.id_num]
+            print('your balance is {}'.format(balance))
+            print('Thank you for using our services\n'
+                  'For this action '
+                  'you will be charged for 300$ commission\n'
+                  'Bye Bye!')
+
+        elif self.choice == DEPOSIT_CHOICE:
+            amount = float(input('How much would you like to deposit?\n'))
```

Comment:
what would happen if the float convention fails? (in case someone enters a string for example?)

Reply:
done